### ISSUE TITLE
remove feature caching capability

### ISSUE THREAD

**Issue by ksens** [Paradigm4/revealgenomics#43]
Need to remove feature caching capability as it is not being used anymore. 

Also the current implementation is leading to errors when new features are being registered (the cache remains stale):

```R
# on a CE system
gh_connect()
init_db(arrays_to_init = get_entity_names(),force = TRUE)
feature_id = register_feature(df = data.frame(name = "dummyFeature",
                                              gene_symbol = "dummySymbol",
                                              featureset_id = 1000,
                                              chromosome="...",
                                              start="...",
                                              end="...",
                                              strand_term = 1,
                                              feature_type = "gene",
                                              source = "...",
                                              stringsAsFactors = FALSE))
# check that get_features() returns 1 row
stopifnot(nrow(get_features()) == 1)

# Now add three more featuers
ftr_id = register_feature(df = data.frame(
      name = c('asdf1', 'asdf2', 'asdf3'), 
      featureset_id = 1, 
      gene_symbol = 'asdf', 
      chromosome = 'X', 
      start = '1', 
      end = '10000', 
      feature_type = 'dummy', 
      source = 'api-test', 
      stringsAsFactors = FALSE))
# check that get_features() returns 4 rows
stopifnot(nrow(get_features()) == 4)
# Error: nrow(get_features()) == 4 is not TRUE
```

Current workaround is as follows, but should not be needed once feature caching is removed

```R
# restart session and run `gh_connect`

# or
scidb4gh:::update_feature_cache()
stopifnot(nrow(get_features()) == 4)
```

**Comment by ksens** [comment]
For reference, noting download times for, and sizes of other entities that are, or have been cached previously. 

```R
# INDIVIDUALS
system.time({i = get_individuals()})
# user  system elapsed 
# 19.727   2.106  29.160 
format(object.size(i), units="Mb")
# [1] "150.9 Mb"

# BIOSAMPLES
system.time({b = get_biosamples()})
# user  system elapsed 
# 20.609   1.438  28.438 
format(object.size(b), units="Mb")
# [1] "186.2 Mb"
```

**Comment by ksens** [comment]
Fixed via PR #46.
